.ci.yaml

```diff
@@ -4317,7 +4317,6 @@ targets:
 
   - name: Mac_arm64_ios run_debug_test_macos
     recipe: devicelab/devicelab_drone
-    presubmit: false # https://github.com/flutter/flutter/issues/118827
```

Comment:
Do we want to make this bringup:true so that it doesn't block the tree on fail initially?

Reply:
Bringup doesn't work in presubmit? This test is already passing successfully in post submit. We're just having it run in post submit now. It was failing before because of the bot, not the test